common/tools/dev-tool/src/commands/run/bundle.ts

```diff
@@ -68,7 +68,7 @@ export default leafCommand(commandInfo, async (options) => {
       "This is probably a mistake. --ignore-missing-node-builtins should only be used with --inject-node-polyfills."
     );
   }
-  if (!browserTest && (polyfillNode || injectNodePolyfills)) {
+  if (!browserTest && injectNodePolyfills) {
```

Comment:
nit: need to update the warning message since the condition changed

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/27377/commits/d72116aa4b20b4a1c4664eb2c55926b3a6d8cb89